app/players/elves/elf.py

```diff
@@ -0,0 +1,15 @@
+from abc import ABC
+
+from app.players.player import Player
+
+
+class Elf(Player, ABC):
```

Comment:
It is not a requirement to explicitly include ABC in this class. It will behave the same with or without this declaration - as an abstract class.